client/src/components/mainContent.jsx

```diff
@@ -151,6 +178,13 @@ const useStyles = makeStyles((theme) => ({
     fontWeight: 500,
     color: "black",
   },
+  DividerInCardContent: {
+    "margin-top" : "30px",
```

Comment:
```suggestion
    marginTop : "30px",
```

to stay consistent with using inline snakeCase key naming convention